api/client/ffs.go

```diff
@@ -160,8 +162,41 @@ func (f *FFS) NewAddr(ctx context.Context, name string, options ...NewAddressOpt
 }
 
 // GetDefaultCidConfig returns a CidConfig built from the default storage config and prepped for the provided cid
-func (f *FFS) GetDefaultCidConfig(ctx context.Context, c cid.Cid) (*rpc.GetDefaultCidConfigReply, error) {
-	return f.client.GetDefaultCidConfig(ctx, &rpc.GetDefaultCidConfigRequest{Cid: c.String()})
+func (f *FFS) GetDefaultCidConfig(ctx context.Context, c cid.Cid) (ff.CidConfig, error) {
+	res, err := f.client.GetDefaultCidConfig(ctx, &rpc.GetDefaultCidConfigRequest{Cid: c.String()})
+	if err != nil {
+		return ff.CidConfig{}, err
+	}
+	resCid, err := cid.Decode(res.Config.Cid)
+	if err != nil {
+		return ff.CidConfig{}, err
+	}
+	return ff.CidConfig{
+		Cid:        resCid,
```

Comment:
We want to return a real cid here so it can be properly marshaled to json by the cli or any other user